pom.xml

```diff
@@ -122,6 +122,18 @@
         </profile>
     </profiles>
     <dependencies>
+        <dependency>
+            <groupId>org.jacoco</groupId>
+            <artifactId>org.jacoco.core</artifactId>
+            <version>0.8.10</version>
```

Comment:
Define version variables instead of fixed values

Reply:
ok